source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -58,13 +58,25 @@
         <div class="title">Plan de acción </div><textarea class="content-editable" data-bind="value: evaluation.actionPlanComment, enable: isEvaluationEditable()"></textarea>
       </div>
     </div>
-    <h4>Comentarios de la Evaluación</h4>
+    <h4>Comentarios de la Empresa</h4>
     <div class="comments-container comments-company-container">
       <textarea class="content-editable big-comment" data-bind="value: generalComment, enable: isEvaluationEditable()"></textarea>
     </div>
-    <h4>Comentarios del Evaluado</h4>
-    <div class="comments-container comments-company-container">
-      <textarea class="content-editable big-comment" data-bind="value: evaluatedComment, enable: evaluation.readyForDevolution && !evaluation.finished && userView == 3"></textarea>
+    <div class="evaluation-comments">
+      <h4>Comentarios de la Evaluación</h4>
+      <div>
+        <!-- ko foreach: califications-->
+        <div data-bind="if: show() && (owner < 2 ||(owner == 2 && comments))">
```

Comment:
Is it the same condition that is used for califications (I mean, numbers)?

In that case what do you think about to move the logic to the view model and remove duplicated code from the view?